ipatests/test_integration/test_subids.py

```diff
@@ -268,3 +269,20 @@ def test_sudid_match_shows_uid(self):
                                           f"--subuid={subuid}"])
         owner = self._parse_result(result)["owner"]
         assert owner == uid
+
+    def test_nsswitch_is_updated_with_subid_entry(self):
+        """
+        This test case checks that when ipa-client-install
+        is installed with --subid option, the nsswitch.conf
+        file is modified with the entry 'subid: sss'
+        """
+        tasks.uninstall_client(self.clients[0])
+        tasks.install_client(
+            self.master, self.clients[0], extra_args=['--subid'],
+            nameservers=None
+        )
```

Comment:
@stanislavlevin in this testcase we are uninstalling the ipa-client and then installing it with --subid option, since the earlier testcase have ipa-client installed on the system.

Reply:
@stanislavlevin, yes that correct what you just mentioned, since num_clients = 1 does the ipa-client install, the newly testcase fails to install ipa-client, hence the tasks.uninstall_client was added and then ipa-client install was done with --subid option which is intentional.